tests/parser.js

```diff
@@ -60,9 +60,8 @@ require([
                         var mug = call("getMugByPath", "/data/state");
                         assert.equal(mug.__className, "Select");
                         var xml = call("createXML"),
-                            doc = $($.parseXML(xml)),
-                            instance = '<instance id="states" src="jr://fixture/item-list:state"></instance>';
-                        assert.operator(xml.indexOf(instance), ">", -1);
+                            doc = $($.parseXML(xml));
+                        assert.equal(doc.find("instance[id=states]").length, 1, xml);
```

Comment:
Fair point. In this case the XML is better than "expected -1 to be > -1" with no other information.